flow/Platform.cpp

```diff
@@ -1835,23 +1944,24 @@ std::string abspath( std::string const& filename ) {
 #endif
 }
 
-std::string basename( std::string const& filename ) {
-	auto abs = abspath(filename);
+std::string parentDirectory( std::string const& path, bool resolveLinks, bool mustExist ) {
+	auto abs = abspath(path, resolveLinks, mustExist);
```

Comment:
Could this be implemented with popPath?

Reply:
I think so, now that absPath() will always remove ".." no matter what.